tests/python_slices/list.py

```diff
@@ -0,0 +1,9 @@
+a[-1]
+a[-2:]
+a[:-2]
+a[::-1]
+a[1::-1]
+a[:-3:-1]
+a[-3::-1]
+point_coords = coords[i, :]
+main(sys.argv[1:])
```

Comment:
Would be nice to see some tests involving wrapping. In particular, I'd like to see where that trailing comma gets placed.